dot/rpc/modules/author_test.go

```diff
@@ -149,11 +149,12 @@ func TestAuthorModule_SubmitExtrinsic_InQueue(t *testing.T) {
 		Validity:  val,
 	}
 
-	txQueue.Push(expected)
+	_, err := txQueue.Push(expected)
+	require.Nil(t, err)
 
 	// this should cause error since transaction is already in txQueue
-	err := auth.SubmitExtrinsic(nil, &ext, res)
-	require.EqualError(t, err, "transaction is already in pool")
+	err = auth.SubmitExtrinsic(nil, &ext, res)
+	require.Nil(t, err)
```

Comment:
This test case is called: TestAuthorModule_SubmitExtrinsic_InQueue but it was expecting a err "transaction is already in pool", but that seemed to be a issue related to the `shadows declaration at line 155 (govet)` issue on `dot/rpc/modules/author.go:172`

Reply:
I'm not sure I understand? this test should return an error